features/users.feature

```diff
@@ -0,0 +1,665 @@
+        "name": "borgana",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "bail.organa@localhost.localdomain",
+        "emails": [{
+          "address": "bail.organa@localhost.localdomain",
+          "active": false
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: c3po",
+        "name": "c3po",
+        "technical": true,
+        "organizationId": "@idOf: Old Republic",
+        "technicalMembership": {
+          "id": "@alphanumeric",
+          "organizationId": "@idOf: Old Republic",
+          "roles": [],
+          "createdAt": "@iso8601",
+          "updatedAt": "@iso8601",
+          "userId": "@idOf: c3po"
+        },
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: dvader",
+        "name": "dvader",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "dark.vader@localhost.localdomain",
+        "emails": [{
+          "address": "dark.vader@localhost.localdomain",
+          "active": false
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: hsolo",
+        "name": "hsolo",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: lskywalker",
+        "name": "lskywalker",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": false,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: master",
+        "name": "master",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": true,
+        "roles": [
+          "admin"
+        ],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: palpatine",
+        "name": "palpatine",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "palpatine@localhost.localdomain",
+        "emails": [{
+          "address": "palpatine@localhost.localdomain",
+          "active": false
+        }],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }, {
+        "id": "@idOf: pamidala",
+        "name": "pamidala",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "primaryEmail": "padme.amidala@localhost.localdomain",
+        "emails": [{
+          "address": "padme.amidala@localhost.localdomain",
+          "active": false
+        } ],
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: Only Probe Dock admins should be able to retrieve all users with only details for organizations where they are member and organization memberships.
```

Comment:
Is this description correct? Doesn't a Probe Dock admin get everything for all organizations?